docs/administration/deployment/deploying-locally/index.md

```diff
@@ -2,7 +2,8 @@
 
 ## For researchers
 
-If you are a researcher and would like to try out deploying a small REANA cluster on your laptop, you can proceed as follows.
+If you are a researcher and would like to try out deploying a small REANA cluster on your laptop,
+you can proceed as follows.
```

Comment:
Minor: What is the reasoning behind this change?

Reply:
It didn't fit on my screen so I trailed it for better readability.